prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Nelson David Daza';
+    console.log(fullName);
+    var numberOfPets = '0';
```

Comment:
Here we want to use an actual number, if we use quotes then it is just a string with a zero inside of it. We want to just use the number literal `0`.

Reply:
Done!

<img width="1440" alt="Screen Shot 2021-09-22 at 3 36 34 PM" src="https://user-images.githubusercontent.com/88061673/134410217-7d094377-9e09-4062-8b88-991d12d4097c.png">